src/style.css

```diff
@@ -0,0 +1,26 @@
+* {
+  margin: 0;
+  background-color: black;
+}
```

Comment:
Try to avoid selector *. Yes in these tasks it doesn't influence performance, but in bigger projects, it can load browser